pybossa/cache/projects.py

```diff
@@ -252,6 +258,11 @@ def search_lock_status_sorting_result():
             results = session.execute(text(sql_query), params)
             tasks = [format_task(row, locked_tasks_in_project.get(row.id, [])) for row in results]
 
+        # format first 100 task dates utc to est
+        # this is to obtain 10x better response time
+        num_tasks = min(len(tasks), 100)
+        for i in range(num_tasks):
```

Comment:
You mentioned using map, we can have ~10% performance improvement. So how about changing the loop by the following:
```
tasks = map(format_task_dates, tasks[:num_tasks]
```
Also, line 268 returns ```tasks``` which can be huge number. Shall we just return the number of tasks at most 100?

Reply:
I didn't found `map` giving significant improvement hence did not considered it. Line 268 has all tasks as task browse records are cached. Thanks!